### PR TITLE
Add a Further Reading section to the README

### PR DESCRIPTION
**Pull request by jenweber**
Partially addresses #286 

The goal is to help demonstrate why inclusive language is important, and help users to notice the ways in which they need thoughtful human editing, beyond `alex`, in order to have inclusive content.

I would like to find an article to replace the SheCanCode one, if anyone has ideas. It leans on an example from US politics, which I would like to avoid, when there are so many other things out there.

Feedback, edits, and additions welcome from anyone who is interested in this.

### CODE REVIEW

**Comment by wooorm** [comment]
nice! I’m assuming the CI complains about line length and list bullets though.

`harver` should perhaps be capitalized?

I’m not sure about an alternative to SheCanCode 🤔

I also recently came across this one, <https://incl.ca/disability-language-is-a-nuanced-thing/>, which might be something you want to add or I can PR later.

**Comment by jenweber** [comment]
Hi @wooorm, thanks for the article suggestion. I included it in the list and replaced the SheCanCode article with a link to Textio. I capitalized `Harver`. I could use your help figuring out what to do with the quotes linting failures. If I swap the regular double quotes `"` for smart quotes, I get:

```
Expected `‘` to be used at this level of nesting, not `“`
```

If I use `‘`, I get:
```
576:61-576:62  warning  Expected `“` to be used at this level of nesting, not `‘`  quote  retext-quote
```

Any ideas?

File: readme.md [hunk 1]
```diff
@@ -557,6 +558,46 @@ It’s a nice unisex name, it was free on npm, I like it!  :smile:
...
+
+*   The [18F Content Guide](https://content-guide.18f.gov/our-style/inclusive-language/)
+    has a helpful list of links to other inclusive language guides used in
+    journalism and academic writing.
+*   The [Conscious Style Guide](https://consciousstyleguide.com/articles/) has
+    articles on many nuanced topics of language.  For example, the terms race
+    and ethnicity mean different things, and choosing the right word is up to
+    you.
+    Likewise, a sentence that overgeneralizes about a group of people
+    (e.g. “Developers love to code all day“) may not be noticed by `alex`, but
```
Comment (wooorm) [inline readme.md:577]:
```suggestion
    (e.g. “Developers love to code all day”) may not be noticed by `alex`, but
```

File: readme.md [hunk 2]
```diff
@@ -557,6 +558,46 @@ It’s a nice unisex name, it was free on npm, I like it!  :smile:
...
+    it is not inclusive.  A good human editor can step up to the challenge and
+    find a better way to phrase things.
+*   Sometimes, the only way to know what is inclusive is to ask.
+    In [Disability is a nuanced thing](https://incl.ca/disability-language-is-a-nuanced-thing/),
+    Nicolas Steenhout writes about how person-first language, such as
+    ”a person with a disability,” is not always the right choice.
```
Comment (wooorm) [inline readme.md:583]:
```suggestion
    “a person with a disability,” is not always the right choice.
```

**Review by wooorm — COMMENTED** [review]
All looks good: the quotes were a bit messed up: closing ones (“99”s) were used to start, and opening ones (“66”s) were used to end.

One more Q: should this be higher? This is good but just thinking about the position, as it’s pretty important info

**Comment by jenweber** [comment]
Sure, I can move this up higher. Thanks for the help with the quote marks.

File: readme.md [hunk 3]
```diff
@@ -557,6 +558,46 @@ It’s a nice unisex name, it was free on npm, I like it!  :smile:
...
+    Nicolas Steenhout writes about how person-first language, such as
+    “a person with a disability,” is not always the right choice.
+*   Language is always evolving.  A term that is neutral one year ago can be
+    problematic today.  Projects like the
+    [Self-Defined Dictionary](https://github.com/selfdefined/web-app) aim to
+    collect the words that we use to define ourselves and others, and connect
+    them with the history and some helpful advice.
+*   Unconsious bias can show up in writing, just like when it is present in
+    daily decisions and conversations.
+    [Textio](https://textio.com/blog/4-overlooked-types-of-bias-in-business-writing/27521593662)
```
Comment (wooorm) [inline readme.md:591]:
```suggestion
*   Unconsious bias is present in daily decisions and conversations and can show
    up in writing.
    [Textio](https://textio.com/blog/4-overlooked-types-of-bias-in-business-writing/27521593662)

```

alex doesn’t like `just`, inverting the sentence prevents it

**Comment by jenweber** [comment]
Hi, I think maybe this is ready to merge now. Tests are green and I moved it above the FAQ. If you want, I can add similar content to the home page. Just let me know if you have any ideas for how you'd want it to look/read in that context.

**Comment by wooorm** [comment]
Oh sorry I missed that, perfect!

**Comment by wooorm** [comment]
This is great, thanks so much @jenweber! 💐

**Pull request merged**